app/qml/ExternalResourceBundle.qml

```diff
@@ -117,7 +123,8 @@ Item {
             var success = __inputUtils.copyFile(imagePath, absolutePath)
             if (!success)
             {
-                console.log("error: Unable to copy file " + imagePath + " to the project directory")
+              errorDialog.errorText = qsTr( "Failed to copy image file to %1. Please ensure that your project configured correctly" ).arg( absolutePath )
```

Comment:
I would remove "Please ensure that your project configured correctly". Also make sure it is logged in console log too!